README.md

```diff
@@ -5,10 +5,57 @@
 
  A wiki that contain all osu! rulesets
 
+## Domain
+
+- Dev build (Some function in dark mode like color invert will not work.) : https://dev.rulesets.info/
+- Official build : https://rulesets.info/
```

Comment:
Official Build: https://rulesets.info/

Is better the way I see it

Reply:
I don't think this needs capitalization like you suggest. This should follow standard casing as it isn't a proper noun.

Reply:
So I'm not change it.